packages/core/README.md

```diff
@@ -45,8 +45,9 @@ the following:
 
 #### Manual mode
 
-Manual mode requires that you pass around the segment reference. See the examples
-in the [express package](https://github.com/aws/aws-xray-sdk-node/tree/master/packages/express) for the different usages.
+Manual mode requires that you pass around the segment reference. See the 
+[manual mode examples](https://github.com/aws/aws-xray-sdk-node/tree/master/packages/core#Manual-Mode-Examples) 
```

Comment:
The link is invalid. I recommend to use https://github.com/aws/aws-xray-sdk-node/tree/master/packages/core#Manual-Mode-Examples